test/RelayHub.test.js

```diff
@@ -0,0 +1,89 @@
+const { BN, ether, expectEvent, time } = require('openzeppelin-test-helpers');
+
+const { getTransactionHash, getTransactionSignature } = require('../src/js/relayclient/utils')
+
+const RelayHub = artifacts.require('RelayHub');
+const SampleRecipient = artifacts.require('SampleRecipient');
+
+contract('RelayHub', function ([_, relayOwner, relay, sender, other]) {  // eslint-disable-line no-unused-vars
```

Comment:
We should be able to use the `varsIgnorePattern` option to avoid doing this, but I couldn't get it to work.